<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpidrDacsDialog</class>
 <widget class="QDialog" name="SpidrDacsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPIDR-DACS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="spidrdacs.qrc">
    <normaloff>:/icon/spidrdacs.ico</normaloff>:/icon/spidrdacs.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxSpidr">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>SPIDR</string>
     </property>
     <widget class="QLabel" name="labelPeriod">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>34</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineEditAddr0">
      <property name="geometry">
       <rect>
        <x>132</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineEditAddr2">
      <property name="geometry">
       <rect>
        <x>56</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>168</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="_pushButtonConnectOrDisconnect">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineEditAddr1">
      <property name="geometry">
       <rect>
        <x>94</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelPeriod_3">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>34</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelIpAddr">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>14</y>
        <width>69</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IP address</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelPort">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>14</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelPeriod_2">
      <property name="geometry">
       <rect>
        <x>84</x>
        <y>34</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineEditAddr3">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>192</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineEditPort">
      <property name="geometry">
       <rect>
        <x>176</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>50000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="_labelDisconnected">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>DISCONNECTED</string>
      </property>
     </widget>
     <widget class="QPushButton" name="_pushButtonReadDacs">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>419</x>
        <y>30</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>Reread DACs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="_pushButtonSetDacsDefaults">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>505</x>
        <y>31</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>Set DACs dflt</string>
      </property>
     </widget>
     <widget class="QComboBox" name="_comboBoxDeviceIndex">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>30</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMonitor">
     <property name="title">
      <string>Timepix3 DACs</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="_gridLayoutDacs">
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="spidrdacs.qrc"/>
 </resources>
 <connections/>
</ui>
